src/styles/main.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./blocks/calendar.scss";
+@import "./blocks/reset.scss";
```

Comment:
these block styles are still missing, unfortunately(
make sure to `git add .` in the root of your directory